### PR TITLE
Remove crash reporting from foss build

### PR DESCRIPTION
**Pull request by mueller-ma**
@digitaldan Can you check if it still works as expected in the full build?

### CODE REVIEW

**Comment by digitaldan** [comment]
Does not look like its working, the only message I get is
```
12-05 16:33:27.120 20945-20945/org.openhab.habdroid I/FirebaseInitProvider: FirebaseApp initialization unsuccessful
```

File: build.gradle [hunk 1]
```diff
@@ -4,18 +4,10 @@ buildscript {
     repositories {
         jcenter()
         mavenCentral()
-        maven {
-            url 'https://maven.google.com'
```
Comment (mueller-ma) [inline build.gradle]: google() is short for maven.google.com

**Comment by mueller-ma** [comment]
@digitaldan Should work now.

**Comment by digitaldan** [comment]
This is what I'm getting now:

```
12-09 10:47:02.549 13724-13724/org.openhab.habdroid I/CrashlyticsInitProvider: CrashlyticsInitProvider initialization successful
12-09 10:47:02.562 13724-13724/org.openhab.habdroid D/FirebaseApp: com.google.firebase.auth.FirebaseAuth is not linked. Skipping initialization.
12-09 10:47:02.592 13724-13724/org.openhab.habdroid D/FirebaseApp: com.google.firebase.crash.FirebaseCrash is not linked. Skipping initialization.
12-09 10:47:02.592 13724-13724/org.openhab.habdroid I/FirebaseInitProvider: FirebaseApp initialization successful
```

I'm not sure if that means its working or not

**Comment by mueller-ma** [comment]
Not sure either. A way to test this: Add something like `int i = 7/0` to mainactivity#oncreate() and look at the platform if the crash shows up.

**Comment by mueller-ma** [comment]
On my phone with play services and current master I see the same:

```
12-21 20:31:36.426 9561-9561/org.openhab.habdroid I/CrashlyticsInitProvider: CrashlyticsInitProvider initialization successful
12-21 20:31:36.436 9561-9561/org.openhab.habdroid D/FirebaseApp: com.google.firebase.auth.FirebaseAuth is not linked. Skipping initialization.
12-21 20:31:36.451 9561-9561/org.openhab.habdroid D/FirebaseApp: com.google.firebase.crash.FirebaseCrash is not linked. Skipping initialization.
12-21 20:31:36.451 9561-9561/org.openhab.habdroid I/FirebaseInitProvider: FirebaseApp initialization successful
```

**Comment by mueller-ma** [comment]
@digitaldan Can you look at this PR?
The second option to remove crashalytics is to create a simple bash script, that removes all non-free libaries from build.gradle.

**Comment by mueller-ma** [comment]
Full build:
```
02-21 20:34:55.447 25513-25513/? I/CrashlyticsCore: Initializing Crashlytics 2.4.1.19
...
02-21 20:34:55.738 25513-25513/org.openhab.habdroid I/CrashlyticsInitProvider: CrashlyticsInitProvider initialization successful
02-21 20:34:55.761 25513-25513/org.openhab.habdroid D/FirebaseApp: com.google.firebase.auth.FirebaseAuth is not linked. Skipping initialization.
02-21 20:34:55.784 25513-25513/org.openhab.habdroid D/FirebaseApp: com.google.firebase.crash.FirebaseCrash is not linked. Skipping initialization.
02-21 20:34:55.784 25513-25513/org.openhab.habdroid I/FirebaseInitProvider: FirebaseApp initialization successful
...
<crash>
02-21 20:34:57.536 25513-25558/org.openhab.habdroid I/CrashlyticsCore: Crashlytics report upload complete: 5A8DC9DF02E7-0001-63A9-18503F83490D

```

In the foss build no log messages related to crashalytics or firebase are there, so it seems good to me.

**Pull request merged**